src/config/main.js

```diff
@@ -5,7 +5,8 @@
 import { jsPsych } from "jspsych-react";
 import _ from "lodash";
 import { eventCodes } from "./trigger";
-import { verifyProlific } from "../lib/utils";
+import { init } from "@brown-ccv/behavioral-task-trials";
```

Comment:
do we really need the behavioral task trials? it doesn't look like we're using any of the trials, so the init won't have an impact